File: ios/RNModuleTemplate.m
//
//  RNModuleTemplate.m
//  RNModuleTemplate
//
//  Copyright © 2020 Alex Demchenko. All rights reserved.
//

#import "RNModuleTemplate.h"

@implementation RNModuleTemplate

RCT_EXPORT_MODULE();

+ (void)hello {}

- (NSDictionary *)constantsToExport
{
  return @{ @"count": @1 };
}

+ (BOOL)requiresMainQueueSetup
{
  return YES;
}

@end

File: ios/RNModuleTemplate.swift
//
//  RNModuleTemplate.swift
//  RNModuleTemplate
//
//  Copyright © 2020 Alex Demchenko. All rights reserved.
//

import Foundation

@objc(RNModuleTemplate)
class RNModuleTemplate: NSObject {
  @objc
  func constantsToExport() -> [AnyHashable : Any]! {
    return ["count": 1]
  }

  @objc
  static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
